src/v2/Components/BankDebitForm/BankDebitForm.tsx

```diff
@@ -45,20 +52,34 @@ export const BankDebitForm: FC<Props> = ({ order }) => {
 
   const handleSubmit = async event => {
     event.preventDefault()
+    setIsInsufficientFunds(false)
+    setIsConfirmingAccountSetup(true)
+    trackClickedContinue()
 
-    if (!stripe || !elements || !user) {
-      return
-    }
+    if (!stripe || !elements || !user) return
 
-    const { error } = await stripe.confirmSetup({
-      elements,
-      confirmParams: {
-        return_url: "https://staging.artsy.net/",
-      },
-    })
+    try {
+      const { setupIntent, error } = await stripe.confirmSetup({
+        elements,
+        redirect: "if_required",
+        confirmParams: {
+          return_url: "https://staging.artsy.net/",
+        },
+      })
+
+      if (setupIntent?.status === "succeeded") {
+        // TODO: poll account balance and set state accordingly
```

Comment:
Based on the [doc](https://stripe.com/docs/payments/ach-debit/set-up-payment?platform=web#access-data-on-a-financial-connections-bank-account), we might want to poll the account balance _before_ we call `confirmSetup`.

> If you opted to receive balances permissions, we recommend checking a balance at this stage to verify sufficient funds before confirming a payment.